spec/lib/gettext_setup_spec.rb

```diff
@@ -50,8 +52,10 @@
       expect(GettextSetup.default_locale).to eq('en')
       expect(GettextSetup.candidate_locales).to include('en')
       GettextSetup.clear
+      before_locale = ENV['LANG']
       ENV['LANG'] = 'de_DE'
       expect(GettextSetup.candidate_locales).to eq('de_DE,de,en')
+      ENV['LANG'] = before_locale
```

Comment:
This was not being unset before, and other tests in the suite were running in de_DE locale, which messes with string change detection because it relies on examing stderr (where the errors were now in German). Whether or not that is itself a problem should probably be discussed, but not cleaning this up seems bad regardless.